lib/thrift/generator/server.ex

```diff
@@ -0,0 +1,143 @@
+defmodule Thrift.Generator.Server do
+  @moduledoc false
+
+  defmodule Framed do
+    @moduledoc false
+
+    alias Thrift.Generator.{
```

Comment:
I dunno if you guys already have an established convention for this, but personally I prefer just listing out all of the aliases explicitly.  That makes it a lot easier to search for `Thrift.Generator.Service`, etc., and it makes it easier to refactor later on.

Disregard if you guys are already going with this style in other places.